### ISSUE TITLE
How on earth to set this up on Docker in 2023

### ISSUE THREAD

**Issue by privacyguy123** [mozilla-services/syncstorage-rs#1483]
```docker-compose``` (python script) is long deprecated replaced by in built ```docker compose``` Instructions extremely unclear, gaps, referencing old and new GitHubs at the same time, "latest" image on Docker Hub actually ancient ... the list goes on?

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/SYNC-3932)

**Comment by jrconlin** [comment]
Hi, so we're a very small team split between several projects. Our main focus is, of course, providing a quality production product for firefox customers. This means that we have to provide a highly scalable system for our primary customers. 

Offering a configuration that can be run on private servers is something that we offered, particularly when we had load that could run on a suite of MySQL clients. That's kind of no longer the case. 

We will do our best to try and support the community, but we need the communities help for that. If there are things that would make your life easier, please do not hesitate to submit a PR. (Check with the [CONTRIBUTING.md](https://github.com/mozilla-services/syncstorage-rs/blob/master/CONTRIBUTING.md) doc to make that smoother.)  

Thanks, as always for helping make things better for everyone.

**Comment by WAdama** [comment]
Have also a look on this thread:
[https://github.com/mozilla-services/syncstorage-rs/issues/1428](url)

**Comment by jeena** [comment]
I crated this from my research for now https://github.com/jeena/fxsync-docker

**Comment by kristof-mattei** [comment]
https://github.com/mozilla-services/syncstorage-rs/blob/68d326701505a7c3ae04d59953eb099cf8add4d2/.circleci/config.yml#L322

Repo only publishes the `spanner` image.

**Comment by WAdama** [comment]
And why the sudden change? 0.13.7 worked with MySQL/MariaDB.

How do I have to change my existing configuration to work with spanner?

**Comment by kristof-mattei** [comment]
> And why the sudden change? 0.13.7 worked with MySQL/MariaDB.
> 
> How do I have to change my existing configuration to work with spanner?
> 
> On the other side, I use selfhosting to get rid of external cloud services, and if I'm not mistaken, Spanner is a Google Cloud service... Hallelujah...

Someone overlooked it? I proposed a fix here: #1496

**Comment by WAdama** [comment]
You're right, I've overseen this totally, but I would second your proposal...

**Comment by clouserw** [comment]
Please find our new documentation on how to deploy with docker at https://mozilla-services.github.io/syncstorage-rs/how-to/how-to-run-with-docker.html .  Thanks
